app/styles/app/modules/build-messages.scss

```diff
@@ -34,14 +34,18 @@
       @include colorSVG($brick-red);
     }
 
+    .icon-alert {
+      @include colorSVG($brick-red);
+    }
+
     .message {
       padding-left: 7px;
       border-left: 1px solid $dry-cement;
       margin-left: 2px;
     }
 
     code {
-      background-color: #eef0f4;
+      background-color: #f4f6fa;
```

Comment:
I personally find this a little harder to read hah (or maybe more distracting?), but maybe that's just me. We've gotten some complaints in the past about low-contrast stuff, so it might be worth running this by the Creative team. What do you think?

I also checked and noticed that we use `#f1f1f1` for some `code` blocks on the repo settings page (eg https://travis-ci.com/travis-ci/travis-web/settings), so that could be another option worth considering.

Reply:
hmmm, maybe. i am guessing you have much better eyes than i do :)

but i'd argue that "low contrast" is not "low contrast", it depends a lot of what being contrasted and why. in this case there's way too many tiny code blocks, so parsing the entire line really hard imo. and that's a case where lower contrast is much better, we still have the different font type as well.

using `#f1f1f1` seems like the path of least conflict then, so i'd use that?

but i can also just revert this if you don't like pushing this without the Creative team being in the loop.

Reply:
Yeah totally, those are good points. I'm actually good with either color, I just wanted to pause a bit, think about it, and get some more of your thoughts on it. :)